src/svg-icons/index-generator.js

```diff
@@ -0,0 +1,38 @@
+const fs = require('fs');
+const rrs = require('recursive-readdir-sync');
+
+var outArray = [];
+outArray.push('module.exports = {\n');
+
+rrs('./').forEach(function(file) {
+	if(file !== 'index-generator.js' && file !== 'index.js')
+	{
+		var fileLines = fs.readFileSync(file, 'utf8').split('\n');
```

Comment:
I think that the eslint task will fail since you are using tab and not space for indent.